src/SqlProcessor.php

```diff
@@ -222,8 +222,10 @@ public function processModifier($type, $value)
 
 					case 'i[]':
 						$i = count($value);
-						while ($i-- && is_int($value[$i]));
-						if ($i >= 0) break; // fallback to processArray
+						foreach ($value as $v) {
+							if (is_int($v)) $i--;
+						}
+						if ($i >= 0) break; // fallback to processArray 
```

Comment:
@hrach hmm why is there `>=`, shouldn't it be just `>`? For `[1]` `$i = 1` and the cycle will be run once so `$i = 0`.